cmd/policy-main.go

```diff
@@ -67,28 +67,31 @@ FLAGS:
   {{end}}{{end}}
 EXAMPLES:
    1. Set bucket to "download" on Amazon S3 cloud storage.
-      $ {{.HelpName}} download s3/burningman2011
+      $ {{.HelpName}} set download s3/burningman2011
 
    2. Set bucket to "public" on Amazon S3 cloud storage.
-      $ {{.HelpName}} public s3/shared
+      $ {{.HelpName}} set public s3/shared
 
    3. Set bucket to "upload" on Amazon S3 cloud storage.
-      $ {{.HelpName}} upload s3/incoming
+      $ {{.HelpName}} set upload s3/incoming
 
-   4. Set a prefix to "public" on Amazon S3 cloud storage.
-      $ {{.HelpName}} public s3/public-commons/images
+   4. Set policy to "public" for bucket with prefix on Amazon S3 cloud storage 
+      $ {{.HelpName}} set public s3/public-commons/images
 
-   5. Set a prefix to the policy file path on Amazon S3 cloud storage.
-      $ {{.HelpName}} /path/to/policy.json s3/public-commons/images
+   5. Set a custom policy to bucket with prefix from a policy file path on Amazon S3 cloud storage.
```

Comment:
Suggest this can be reworded to
`Set a custom prefix based bucket policy on Amazon S3 cloud storage`